configuration/pih/scripts/global/htmlFormFunctions.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Adds javascript functions for use within html forms
+ * Requires jQuery and moment js
+ */
+class HtmlFormFunctions {
```

Comment:
Is this mean to replace/be an improvement to stuff currently offered by JS code within the HFE module itself?

Reply:
Such as what @mogoodrich ?

Reply:
See the getField, getValue and setValue as defined here: https://wiki.openmrs.org/display/docs/HTML+Form+Entry+JavaScript+Reference

(which admittedly would't handle patientUuid and encounterUuid, I believe)

Reply:
I believe those functions only work in edit/enter modes, right @mogoodrich ?  I'm fine moving away from our custom functions to ones that are in HFE, but the fact is that our HTML Forms are littered with code that gets encounter date in all sorts of custom and different ways, and often doesn't get it using these hfe functions.  So we'd want to have tickets specifically created to thoroughly review particular use cases and handle them appropriately.

Reply:
Ah, it's possible the don't work in view mode, fair enough.... 

Agree that our code is likely littered with code that gets dates in different ways, and probably doesn't use the HFE functions... not saying we need to clean this all up now, just didn't want to start propagating another method... but, yeah, if those getter/setters don't work in View mode and we need something new anyway, better to set up a standard...